omero/sysadmins/unix/server-ubuntu1604-ice36.rst

```diff
@@ -9,7 +9,7 @@ a dedicated local system user, and should be read in conjunction with
 :doc:`install-web`. You can use this as a guide
 for setting up your own test server. For production use you should also read
 the pages listed under :ref:`index-optimizing-server`.
-This guide will install Python 3.5.
+This guide will install Python 3.5 or Python 3.6.
```

Comment:
This might be a bit unclear. Is it "will allow you to choose 3.5 or 3.6" or "shows how to install both" ...

Reply:
You can have both installed. The idea is to show how to install it. The user can choose what he/she wants. I did not want to have another page with few adjustments solely for python

Reply:
That's fine. I simply mean this sentence-- it parses oddly. As in, you won't know which one you are going to get.